### PR TITLE
Chad theme switcher

### PR DESCRIPTION
**Pull request by G-Rowell**
# Hi all,

firstly....a whole lot of work within this PR was done by @Akianonymus, so a huge thanks to them!
Have to stress that this was a joint PR! :heart:

This PR introduces a Telescope theme switcher, which is compatible with NvChad. 
Critically this means that it supports all of the colours that NvChad supports (for proper plugin colour configuration).

Running the colorscheme changer in NVim: `<LEADER> + t + h`

# **NOTE**
This *is a WIP*, so if it is merged there will likely need to be another PR to finish this off the functionality that is planned.
## *Roadmap:*
- A confirm prompt to set this as the default colourscheme (keep it after NVim restart)
- Proper integration with the planned custom config part of NcChad (for the previous dot point)
- Modification of the UI & various Telescope polishes

Thanks, G-Rowell & Akinonymous,

*tl;dr* monkey changes colours nicely!

### CODE REVIEW

**Comment by G-Rowell** [comment]
@siduck76 @Akianonymus 

Definitely fair to say Akianonymus has taken over the work and done a whole lot here!

**Comment by siduck76** [comment]
> @siduck76 @Akianonymus
> 
> Definitely fair to say Akianonymus has taken over the work and done a whole lot here!

yes Im aware! thats! He has been helping a lot lately , a nice NvLad i see

**Comment by G-Rowell** [comment]
@siduck76 Fix merge conflict

**Comment by siduck76** [comment]
> @siduck76 Fix merge conflict

Aki is going to add write the theme to the file too , let's wait

**Comment by Akianonymus** [comment]
@G-Rowell Don't do that, reset, pull the upstream, pick  your commits, then push
